hoodie-hadoop-mr/src/test/resources/sample1.avro

```diff
@@ -1,17 +1,23 @@
 {
```

Comment:
Reuse the test schema under SchemaTestUtils and we have a helper method to add hoodie specific fields to the schema